spec/features/generic_announcements_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+
+RSpec.describe 'Generic announcements', type: :feature do
```

Comment:
See #306 for the kind of feature specs... this builds on top of the fact that we want to structure our specs as user stories.